### ISSUE TITLE
Postman Proxy to can't capture https requests with port other than 443

### ISSUE THREAD

**Issue by wesleyel** [a85/POSTMan-Chrome-Extension#10886]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When I use a proxy to capture Android application traffic, https requests to ports other than the default 443 do not appear in the result list.

Only find `https://example.com/api/endpoint?query=example`,

but no `https://example2.com:9091/api/endpoint?query=example`

At the same time, the Android program that send the request also received no response.

Is it a bug or a a feature that I didn't set correctly.

### Steps To Reproduce

1. open capture proxy tab and set port to 8080
2. install Android certification and setup wifi proxy
3. open App and start capturing
4. problem showing

### Operating System

macOS

### Postman Version

9.14.0

### Postman Platform

Postman App

**Comment by DefCon-007** [comment]
Hi @magicwenli! 

Thank you for reporting this, can you please confirm that without proxy, the https requests to port other than 443 are working on Android? 

Also, can you once try out the same URL on desktop once with and without proxy and check if it works? This will allow us scope the issue better. 

Thanks.

**Comment by wesleyel** [comment]
I used `mkcert` to test the local server, but the proxy program cannot connect https traffic other than 443 port.

![image](https://user-images.githubusercontent.com/48174882/167090004-bdccd4ef-470d-47b1-a59c-40611c07199c.png)

Here's my test file, need to add '127.0.0.1 postman.test' to `/etc/hosts`

[postman-test.zip](https://github.com/postmanlabs/postman-app-support/files/8638657/postman-test.zip)

Thanks for your reply.

**Comment by postman-staff** [comment]
Hi there,
Thanks a lot for attaching the file that you used to test this.

We will try to replicate the behavior and reach you back with an update.
via @nasco-a | Postman Support

**Comment by DefCon-007** [comment]
Hi @magicwenli

Thanks a lot for providing the detailed steps, it looks like `mkcert` uses its own local root CA certificate for signing `postman.test` domain and when Postman receives the request from that domain it is unable to create a new SSL certificate for it and hence it is not able to capture the request. 

We found that this behaviour is replicable with other proxy clients like mitmproxy too where the local https request for signed certificates fail. 

We have registered this feature request internally. We will update this thread when it is released.

Thanks again for flagging this and providing detailed steps to reproduce.
